internal/generator/fluentd/elements/stdout.go

```diff
@@ -7,3 +7,19 @@ const ToStdOut = `
  @type stdout
 </match>
 {{end}}`
+
+type StdOutFilter struct {
+	Pattern string
+}
+
+func (c StdOutFilter) Name() string {
+	return "filterToStdout"
+}
+
+func (c StdOutFilter) Template() string {
```

Comment:
Good idea for tests. 
@Pranjal-Gupta2 - this would be ideal for your rate limiting tests, you can attach the test to vector's stdout and get exact time of arrival of messages without loki or other store in the way. No need to poll for metrics, you can measure with time.Now() directly.  It can be a functional test that needs nothing but the vector collector.

Reply:
@jcantrill are  you planning to expose this as an output or is there some other way to enable it for tests?
We'd need it for vector also.

Reply:
> @jcantrill are you planning to expose this as an output or is there some other way to enable it for tests? We'd need it for vector also.

I used this in testing an manually updated the config generation.  I had no intention to expose as output.  I think it would be clever is there were a way to expose it in a "dev" mode.  Maybe via golang tag? that makes it a viable output type?